exporter/qrynexporter/logs.go

```diff
@@ -430,7 +432,13 @@ func (e *logsExporter) pushLogsData(ctx context.Context, ld plog.Logs) error {
 		}
 	}
 
-	return batchSamplesAndTimeSeries(context.WithValue(ctx, "cluster", e.cluster), e.db, samples, timeSeries)
+	if err := batchSamplesAndTimeSeries(context.WithValue(ctx, "cluster", e.cluster), e.db, samples, timeSeries); err != nil {
+		return err
+	}
+
+	e.logger.Info("pushLogsData", zap.Int("samples", len(samples)), zap.Int("timeseries", len(timeSeries)), zap.String("cost", time.Since(start).String()))
```

Comment:
Do we need that in the Info level? May Debug be more appropriate?

Reply:
Oh, sure. I just copied whatever we already have for [traces](https://github.com/metrico/otel-collector/blob/c1c0373134881b20a88a939a7afea4556a8a6956/exporter/qrynexporter/traces.go#L188). I can update this and change all to Debug.

Reply:
Thanks. Please change.